src/lib/utils.ts

```diff
@@ -417,3 +417,14 @@ export const getOrderItemQuantities = (items: PublicOrderItem[]): PublicOrderIte
 
   return Array.from(itemMap.values());
 };
+
+/**
+ * Validates src for event cover image, returning a default if invalid
+ * @param src src for cover image
+ * @returns a valid image src
+ */
+export const getDefaultEventCover = (src: any): string => {
```

Comment:
```suggestion
export const getDefaultEventCover = (src: unknown): string => {
```

to make sure you're checking for all possible types exhaustively